docs/articles/new-expensify/chat/Start-a-conversation-thread.md

```diff
@@ -0,0 +1,33 @@
+---
+title: Start a conversation thread
+description: Start a private conversation related to a different message
+---
+<div id="new-expensify" markdown="1">
+
+You can respond directly to a message sent in a chat group or room to start a private 1-on-1 chat with another member about the message (instead of replying to the entire group or room). 
```

Comment:
Hmm not a blocker but `private` suggests to me that the threaded convo can't be seen by others, which isn't true